tests/test_distances.py

```diff
@@ -39,30 +40,17 @@ def test_haversine():
     assert distance == pytest.approx(distance_expected, rel=0.01)
 
 
-@pytest.mark.xfail(reason="API Key issues")
-def test_geocoding_airport_FRA():
+def test_geocoding_airport_FRA(mocker):
```

Comment:
I have a general question regarding this script. It seems like I don't understand how the testing works. Where are the functions in this script called?

Reply:
The test functions itself? They are called when running `pytest`. `pytest` recognizes all tests if their name starts with `test` and then runs them.
The functions to be tested are usually called shortly before the `assert` statements of the `with pytest.raises(Exception)` statements.